website/content/en/status/report-2023-01-2023-03/openstack-on-freebsd.adoc

```diff
@@ -0,0 +1,37 @@
+=== OpenStack on FreeBSD
+
+Links: +
+link:https://www.openstack.org/[OpenStack] URL: link:https://www.openstack.org/[] +
+link:https://github.com/openstack-on-freebsd[OpenStack on FreeBSD] URL: link:https://github.com/openstack-on-freebsd[] +
+
+Contact: Chih-Hsin Chang <starbops@hey.com> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+This project aims to port key OpenStack components so that FreeBSD can function as an OpenStack host.
+
+In 2023 Q1, the big news is that we're able to spawn FreeBSD instances with `bhyve` on the OpenStack platform.
+But there are still some major limitations regarding the capabilities of the spawned instances that need to be resolved:
+
+* No self-service networks (only support the flat network)
+* No network connectivity inside the instance
+* Only support FreeBSD raw images (`FreeBSD-13.1-RELEASE-amd64.raw` tested)
+* No disk resize
+* No console integration (need to use `cu` command manually)
```

Comment:
Same as above: `man:cu[1]`.

Reply:
Updated and please take a look. Thank you.